packages/app-bridge/src/tests/PortalNavigationTreeDummy.ts

```diff
@@ -69,7 +69,7 @@ class GuidelineDocumentGroupClass implements GuidelineDocumentGroup {
     }
 }
 
-class GuidelineDocumentClass implements GuidelineDocument {
+export class GuidelineDocumentClass implements GuidelineDocument {
```

Comment:
Interesting, should those test types be exposed? I take it on my list.

Reply:
In the AppBridge we export all the Dummy types for tests. Maybe we can change that in future, if not really needed.